KsApi/models/CommentTests.swift

```diff
@@ -0,0 +1,20 @@
+@testable import KsApi
+import XCTest
+
+final class CommentTests: XCTestCase {
```

Comment:
is this necessary, because we test it as part of the `postCommentSuccessful` test in the `CommentsViewModel`. It has to deserialize the optimistic comment and compare it to what is in the `.template` no?

Reply:
Because we're manually creating a new comment object, this is to ensure the comment is created with the right values based on what is supplied i.e. `(project, user, body)`